src/main/java/com/digitax/controller/AuthController.java

```diff
@@ -125,8 +125,9 @@
         User user = new User(signUpRequest.getUsername(),
```

Comment:
@Jayanta-Redapple [minor] [optional] signin and signup are one word. So for camelCasing, it would be like signinRequest and signupRequest (signoutRequest). We can keep as two words for now. So long that we are consistent in code, this works for me.